src/renderer/powercord/stylemanager.ts

```diff
@@ -0,0 +1,245 @@
+import AddonPanel from "@ui/components/addonpanel";
+import {DataStore, DiscordModules} from "@modules";
+import SettingsRenderer from "@modules/settings";
+import LoggerModule from "@modules/logger";
+import {fs, Module, path, require as Require} from "@node";
+import Emitter from "../classes/staticemitter";
+import Theme from "@powercord/classes/theme";
+
+const Logger = LoggerModule.create("StyleManager");
+
+export default class StyleManager extends Emitter {
+    static get folder() {return path.resolve(DataStore.baseDir, "themes")};
+
+    static mainFiles = ["powercord_manifest.json", "manifest.json"];
+
+    static themes = new Map();
+
+    static states: object;
+
+    static get addons() {return Array.from(this.themes, e => e[1]);}
+
+    static initialize() {
+        SettingsRenderer.registerPanel("themes", {
+            label: "Themes",
+            order: 1,
```

Comment:
Please change the tab order to `2` since that's the proper order in the settings sidebar. Don't forget to update it at https://github.com/strencher-kernel/pc-compat/blob/dev/src/renderer/ui/quickcss/index.ts#L24 to 3 as well.